Sources/SotoCore/Doc/Region.swift

```diff
@@ -167,3 +167,70 @@ extension AWSPartition {
         }
     }
 }
+
+// allows to create a Region from a String
+// this initialializer returns an Optional.
+// it is non nil for valid region name only
+public extension Region {
+    init?(regionName: String) {
```

Comment:
Can we change the parameter name to `awsRegionName`. This initializer is only needed if you want to verify the region is an AWS region

Reply:
done